ampl-converter/src/main/java/com/powsybl/ampl/converter/version/ExtendedAmplExporterV1.java

```diff
@@ -0,0 +1,335 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.ampl.converter.version;
+
+import com.powsybl.ampl.converter.AmplConstants;
+import com.powsybl.ampl.converter.AmplExportConfig;
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.ampl.converter.AmplUtil;
+import com.powsybl.ampl.converter.util.NetworkUtil;
+import com.powsybl.commons.io.table.Column;
+import com.powsybl.commons.io.table.TableFormatter;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.util.ConnectedComponents;
+
+import java.io.IOException;
+import java.util.List;
+
+import static com.powsybl.ampl.converter.AmplConstants.*;
+
+/**
+ * 1st (V1) extension of BasicAmplExporter, associated with AMPL version 1.1 (exporter id)
+ *
+ * @author Nicolas PIERRE {@literal <nicolas.pierre at artelys.com>}
+ */
+public class ExtendedAmplExporterV1 extends BasicAmplExporter {
+
+    public ExtendedAmplExporterV1(AmplExportConfig config,
+                                  Network network,
+                                  StringToIntMapper<AmplSubset> mapper,
+                                  int variantIndex, int faultNum, int actionNum) {
+        super(config, network, mapper, variantIndex, faultNum, actionNum);
+    }
+
+    private static class ImpedanceAndAdmittance {
+        double r;
+        double x;
+        double g;
+        double b;
+
+        public ImpedanceAndAdmittance(double r, double x, double g, double b) {
+            this.r = r;
+            this.x = x;
+            this.g = g;
+            this.b = b;
+        }
+    }
+
+    @Override
+    public List<Column> getBusesColumns() {
+        return List.of(new Column(VARIANT),
+            new Column(NUM),
+            new Column(SUBSTATION),
+            new Column("cc"),
+            new Column("v (pu)"),
+            new Column("theta (rad)"),
+            new Column("p (MW)"),
+            new Column("q (MVar)"),
+            new Column("slack bus"),
```

Comment:
Code duplication can be avoided in defining a method which add this `Column` in `ExtendedAmplExporterV1` and do nothing in `BasicAmplExporter`.

Same mechanism can be used for the other `getXxxColumns` methods.

Reply:
Thank you for the remark @olperr1. I changed the duplication.